tests/utils/test_logger.py

```diff
@@ -0,0 +1,44 @@
+from io import StringIO
+import sys
+from launchable.utils.logger import Logger
+import launchable.utils.logger as logger
+from unittest import TestCase
+from unittest.mock import patch
+import logging
+
+
+class LoggerTest(TestCase):
+    default_root_handler = None
+
+    def setUp(self):
+        self.default_root_handler = logging.root
+
+        # reset logging handelr for test
+        for handler in logging.root.handlers[:]:
+            logging.root.removeHandler(handler)
+
+    def tearDown(self):
+        logging.root = self.default_root_handler
```

Comment:
I guess this line can't reset the root handler because you copied the root's reference. `self.default_root_handler` may be removed by the above lines. You should deep copy `logging.root` or `logging.root.handlers`.

Reply:
thank you, I'll check it